app/src/main/java/com/morihacky/android/rxjava/fragments/DoubleBindingTextViewFragment.java

```diff
@@ -36,34 +41,80 @@ public View onCreateView(LayoutInflater inflater,
         View layout = inflater.inflate(R.layout.fragment_double_binding_textview, container, false);
         ButterKnife.bind(this, layout);
 
+        if (usePattern1) {
```

Comment:
hmm.. can you explain the purpose of each pattern? i'm inclined to not have this, since it's more of a "flourish" to the core example. 

objective is to only demonstrate the case. the hope is, with a basal understanding people can get figure out variations from that.